standalone-haddock.cabal

```
@@ -23,12 +23,13 @@ executable standalone-haddock
     Cabal.Simple
   -- other-extensions:    
   build-depends:
-    base == 4.*,
-    Cabal >= 1.22,
+    base >= 4.7 && < 5,
```

```suggestion
    base >= 4.11 && < 5,
```

It appears that 4.11 is the minimum required because of the use of Semigroup.